tests/install_upgrade_operators/crypto_policy/test_pqc_tls_audit.py

```
@@ -0,0 +1,91 @@
+        """Verify every CNV service negotiates PQC key exchange.
+
+        Probes each service with multiple PQC groups (X25519MLKEM768, SecP256r1MLKEM768,
+        SecP384r1MLKEM1024) and passes if any group negotiates successfully.
+        On FIPS clusters, all services must reject PQC (ML-KEM not FIPS-certified).
+        """
+        for service_name, accepted in pqc_status_by_service.items():
+            with subtests.test(msg=service_name):
+                if service_name == HYPERCONVERGED_CLUSTER_CLI_DOWNLOAD:
+                    pytest.xfail("Service serves plaintext HTTP behind TLS route, TLS planned for OCP 5.0")
+                if jira_id := SERVICES_WITH_OPEN_BUGS.get(service_name):
+                    if is_jira_open(jira_id=jira_id):
```

Ruff (0.15.12)

[warning] Use a single `if` statement instead of nested `if` statements

(SIM102)